lib/metadataTypes/Interaction.js

```diff
@@ -21,16 +21,59 @@ class Interaction extends MetadataType {
      * @returns {Promise.<TYPE.MetadataTypeMapObj>} Promise
      */
     static retrieve(retrieveDir, _, __, ___, key) {
+        let singleKey = '';
+        if (key) {
+            /* eslint-disable unicorn/prefer-ternary */
+            if (key.startsWith('id:')) {
+                // ! allow selecting journeys by ID because that's what users see in the URL
+                singleKey = '/' + encodeURIComponent(key.slice(3));
```

Comment:
id should not need encoding, what they see on the screen is what is needed by api.

Reply:
good point